.github/workflows/pd-tests.yaml

```diff
@@ -72,7 +72,7 @@ jobs:
           TOTAL_JOBS: ${{needs.chunks.outputs.job-total}}
         run: for i in $(seq 1 $TOTAL_JOBS); do cat covprofile_$i >> covprofile; done
       - name: Send coverage
-        uses: codecov/codecov-action@v1
+        uses: codecov/codecov-action@v4.2.0
```

Comment:
error is 
```bash
https://codecov.io/upload/v4?package=github-action-v1.5.2-1.0.3&token=secret&branch=improve-linter&commit=9f34462e696c925657636f437c894bad31fac16e&build=8586757005&build_url=http%3A%2F%2Fgithub.com%2Ftikv%2Fpd%2Factions%2Fruns%2F8586757005&name=codecov-umbrella&tag=&slug=tikv%2Fpd&service=github-actions&flags=unittests&pr=8015&job=PD+Test&cmd_args=n,F,Q,f,C
{'detail': ErrorDetail(string='Unable to locate build via Github Actions API. Please upload with the Codecov repository upload token to resolve issue.', code='not_found')}
```

need to update codecov version ref https://github.com/codecov/codecov-action/issues/557

can check more information in https://github.com/tikv/pd/actions/runs/8586757005/job/23529728059?pr=8015